src/applications/facility-locator/tests/components/__snapshots__/SearchResultMessage.unit.spec.jsx.mocha-snapshot

```diff
@@ -296,14 +296,14 @@ exports["SearchResultMessage/Should render SearchResultMessage with location err
         "nodeType": "host",
         "type": "p",
         "props": {
-          "children": "Something’s not quite right. Please enter a valid or different location and try your search again."
+          "children": "We’re sorry. Something went wrong on our end. Please try again."
         },
-        "rendered": "Something’s not quite right. Please enter a valid or different location and try your search again."
+        "rendered": "We’re sorry. Something went wrong on our end. Please try again."
       },
       "type": "p",
       "props": {},
       "children": [
-        "Something’s not quite right. Please enter a valid or different location and try your search again."
+        "We’re sorry. Something went wrong on our end. Please try again."
       ]
     },
     {
```

Comment:
Do you feel these snapshot tests in `src/applications/facility-locator/tests/components/SearchResultMessage.unit.spec.jsx` are providing value? Maybe we should change them to look for specific DOM elements instead.